test/Sentry.Tests/ApiApprovalTests.Run.DotNet7_0.verified.txt

```diff
@@ -371,6 +388,17 @@ namespace Sentry
         public void WriteTo(System.Text.Json.Utf8JsonWriter writer, Sentry.Extensibility.IDiagnosticLogger? logger) { }
         public static Sentry.Request FromJson(System.Text.Json.JsonElement json) { }
     }
+    public sealed class ResponseContext : Sentry.IJsonSerializable
+    {
+        public ResponseContext() { }
+        public long? BodySize { get; set; }
+        public string? Cookies { get; set; }
+        public System.Collections.Generic.IDictionary<string, string> Headers { get; }
```

Comment:
@mattjohnsonpint I see `VerifyExtensions.IgnoreStandardSentryMembers` ignores the Dictionary properties of the Request class. I wondered if we should be doing the same for ResponseContext.Headers... but I'm not familiar with Verify so wanted to understand why we'd want to do this before configuring additional `IgnoreMembers` in there.

Reply:
We should only ignore members that vary significantly every time they're run, or vary when run on different machines.  The point is to have snapshots that work everywhere.  If they already do, then no need to add more ignores.